include/validator/formchecker.h

```diff
@@ -36,7 +36,29 @@ enum class VType : uint8_t {
   F64,
   V128,
   FuncRef,
-  ExternRef
+  ExternRef,
+  Bool,
```

Comment:
Not sure if we should export all interface types in this layer, or if we can have a second layer such as `VType::InterfaceType::Bool` to handle these types. I will keep a space to discuss this design.